resources/airbnb.swiftformat

```diff
@@ -59,6 +59,10 @@
 --rules wrap
 --rules wrapArguments
 --rules wrapAttributes
+--rules wrapConditionalBodies
```

Comment:
I'm confused because this is what I'm getting with the current Slumber rules. Do you have the latest SwiftFormat as mentioned in the file (0.50-beta-1)
<img width="846" alt="Screenshot 2022-04-21 at 13 55 14" src="https://user-images.githubusercontent.com/11987427/164452879-e0ac2d27-578c-4d2e-9506-e09ddd499eac.png">
Btw. I personally find it harder to read when the braces are on new lines, also it's a bit confusing that for non-multiline conditions they are on new lines. But I can live with it.
Anyway ,please check if you have the latest SwiftFormat required by the rules we are currently using as for me they seem to be already infered without the need to explicitly putting them here.

Reply:
Yes, I have checked, and the `wrapMultilineStatementBraces` rule is definitely missing from Airbnb's swift format file.

Isn't the example you're showing old code? From when we used to have this rule? I just checked out BTF and tried adding new, unformatted code:
![Capture d’écran 2022-04-21 à 09 37 16](https://user-images.githubusercontent.com/75146312/164470063-b6a9a5dc-f260-421b-a35a-b9bdf66c0dc8.png)
And after running Swift Format:
![Capture d’écran 2022-04-21 à 09 36 52](https://user-images.githubusercontent.com/75146312/164470119-c8f16c7b-6d17-4b48-9669-6dc9025528de.png)

The brace is not moved to the line below as you can see, so it seems to have the issue I was mentioning, which would be normal since the rule is not in the .swiftformat file.